client/outfits.lua

```diff
@@ -50,28 +50,36 @@ function LoadJobOutfit(oData)
         SetPedComponentVariation(ped, 6, data["shoes"].item, data["shoes"].texture, 0)
     end
 
-    -- Bag
-    if data["bag"] ~= nil then
-        SetPedComponentVariation(ped, 5, data["bag"].item, data["bag"].texture, 0)
-    end
-
     -- Badge
     if data["decals"] ~= nil then
         SetPedComponentVariation(ped, 10, data["decals"].item, data["decals"].texture, 0)
     end
 
     -- Accessory
     if data["accessory"] ~= nil then
+        local config = GetDefaultConfig()
+        local tracker = config.trackerClothingOptions
+
         if Framework.HasTracker() then
-            SetPedComponentVariation(ped, 7, 13, 0, 0)
+            SetPedComponentVariation(ped, 7, tracker.item, tracker.texture, 0)
         else
             SetPedComponentVariation(ped, 7, data["accessory"].item, data["accessory"].texture, 0)
         end
     else
+        local config = GetDefaultConfig()
```

Comment:
Same here. You do not need to get the config here since that is strictly for the UI. Just use `Config.TrackerClothingOptions`